pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -579,6 +579,20 @@ self: super: {
     dependencies = with super; [ bclose-vim ];
   });
 
+  telescope-fzy-native-nvim = super.telescope-fzy-native-nvim.overrideAttrs(old: {
+    # it already contains precompiled libraries
+    # rm them and build our own
+    preFixup = ''
```

Comment:
`make` should be invoked in the the build phase, it looks a bit strange to call it in the fixup phase: https://nixos.org/manual/nixpkgs/stable/#ssec-fixup-phase